docs/next/guides/building-and-testing/modules.md

```diff
@@ -481,352 +275,822 @@ new Handsontable(container, {});
+registerLanguageDictionary(plPL.languageCode, plPL);
 ```
 
-Now, you can use newly registered translation. The full example looks like this:
+#### Step 5: Use your translation
+
+Now, you can start using your translation, by setting the [`language`](@/api/options.md#language) [configuration option](@/guides/getting-started/setting-options.md):
 
 ```js
 import Handsontable from 'handsontable/base';
+import 'handsontable/dist/handsontable.full.css';
 import { registerLanguageDictionary, plPL } from 'handsontable/i18n';
 
-registerLanguageDictionary(plPL);
+registerLanguageDictionary(plPL.languageCode, plPL);
 
-// use the locales
+// set the `language` configuration option
 new Handsontable(container, {
   language: 'pl-PL',
-// rest of the settings
 });
 ```
 
-And that is all! You can use the PL-pl translation!
+:::tip
+#### Optimizing Moment.js locales
 
-### Optimizing moment.js locales
+By default, [Moment.js](https://momentjs.com/) (Handsontable's dependency) comes with all possible locales, which increases the bundle size.
 
-If you want to decrease the bundle size even more you can also focus on optimizing moment.js locales. There are different methods of doing so but this guide focuses on using the webpack's [IgnorePlugin](https://webpack.js.org/plugins/ignore-plugin/). For another option, you can check [this tutorial](https://github.com/jmblog/how-to-optimize-momentjs-with-webpack) directly.
-
-Moment.js can be heavyweight because when you type `var moment = require('moment')` in the code you get all locales as default. To be more selective with your choice you can first, use the `IgnorePlugin` that removes all locales:
+To [optimize Moment.js locales](https://github.com/jmblog/how-to-optimize-momentjs-with-webpack), use [webpack's `IgnorePlugin`](https://webpack.js.org/plugins/ignore-plugin/):
 
 ```js
 const webpack = require('webpack');
 module.exports = {
   //...
   plugins: [
-    // Ignore all locale files of moment.js
+    // ignore all Moment.js locale files
     new webpack.IgnorePlugin(/^\.\/locale$/, /moment$/),
   ],
 };
 ```
 
-And then explicitly load selected locales:
+And then explicitly load Moment.js, importing only those locales that you need:
 
 ```js
-import moment from 'moment';
-import 'moment/locale/ja';
 import Handsontable from 'handsontable/base';
+import 'handsontable/dist/handsontable.full.css';
 import { registerCellType, DateCellType } from 'handsontable/cellTypes';
 
+// explicitly import Moment.js
+import moment from 'moment';
+// explicitly import a Moment.js locale of your choice
+import 'moment/locale/ja';
+
+// register the Moment.js locale of your choice
 moment.locale('ja');
 registerCellType(DateCellType);
 
 new Handsontable(container, {
   type: 'date',
 });
 ```
+:::
 
-## Using modules with frameworks
+### Using modules with framework wrappers
 
-You can also use modules with Handsontable's framework wrappers:
+You can use modules with Handsontable's framework wrappers:
```

Comment:
How about:

> If you are using one of the Handsontable's framework wrappers, you might find additional information for your framework on the following pages:

Reply:
@warpech - to be honest, using conditionals, continuous tenses and modal verbs in docs is usually a no-no (unless necessary). I'm not saying we should strictly stick to [STE](https://en.wikipedia.org/wiki/Simplified_Technical_English) or sth similar, but I believe simplicity is more important than likeability of tone. My proposition:

> To find out how to use modules with Handsontable's framework wrappers, see the following guides:

Of course, it's ultimately an arbitrary style guide decision (and we don't have a style guide), so it's up to you :)

Reply:
Your proposal is much better than mine and conveys everything I wanted. Thanks!